### DISCUSSION TITLE
Support for data migration

### DISCUSSION THREAD

**Discussion by abriginets** [planetscale/discussion#92] [Feature Requests] [answered]
Hello. Do you guys plan to support data migration as well for PlanetScale database? I would love to migrate schema changes along with data so that I won't need to do it manually using scripts or whatever.

**Comment by mscoutermarsh** [comment, marked as answer]
Hi @abriginets. This is something we're looking into building in the future.

Could you share more about your current process?

**Reply by abriginets** [reply]
This is pretty simple process
1. Create a `.ts` migration file (unix timestamp being prepended to the filename automatically) via shell command
2. Write migration using query builder (we use Knex)
3. Run migrations against prod on CI
